### ISSUE TITLE
do you have a docker hub image?

### ISSUE THREAD

**Issue by colzhku** [Aidaho12/haproxy-wi#179]
do you have a docker hub image? i need to run it on docker swarm.

**Comment by Aidaho12** [comment]
Yes, I do. But it works as well as rpm, I mean subscribe is need. You can read about it here: https://haproxy-wi.org/installation.py

**Comment by doughnet** [comment]
There is no way to register without paying a donation?

That is not what your documentation states.

"Before uses registry you **should** donate to support project..."

That statement means it isn't a requirement. So how does one use docker without donating?  If you are requiring donations to use your project I definitely will not use it.

**Comment by Aidaho12** [comment]
You can build your own image, there is full instruction how it run manually, just run it in Dockerfile. 

Or if you are student or you want to use it for non-commercial purposes, I can provide it for free

**Comment by doughnet** [comment]
Thank you for the response.  That did not answer my question on why it is required to donate to use it when it does not specifically state that.  I feel that is very deceiving to not state that the software requires payment before wasting a lot of people's time.  If you want to allow student/non-commercial users to access it for free make a method to do that automated.  Seems like a good project but not if it's going to be a paid software when there is Nginx Proxy Manager (NPM) for free available and have an easy docker setup!

**Comment by Aidaho12** [comment]
Because I'm too polite, and I can't write "must" :)

Use Nginx proxy manager, what for this all? You just want to say your "feee"? ) It's my project and my rules, sorry for it :)
